src/nvim/main.c

```diff
@@ -1837,16 +1837,9 @@ static void source_startup_scripts(mparm_T *parmp)
      * - second user exrc file ($VIM/.exrc for Dos)
      * The first that exists is used, the rest is ignored.
      */
+    char_u *user_vimrc = (char_u *)get_from_user_conf("init.vim");
     if (process_env("VIMINIT", true) != OK) {
-      if (do_source((char_u *)USR_VIMRC_FILE, TRUE, DOSO_VIMRC) == FAIL
-#ifdef USR_VIMRC_FILE2
-          && do_source((char_u *)USR_VIMRC_FILE2, TRUE,
-            DOSO_VIMRC) == FAIL
-#endif
-#ifdef USR_VIMRC_FILE3
-          && do_source((char_u *)USR_VIMRC_FILE3, TRUE,
-            DOSO_VIMRC) == FAIL
-#endif
+      if (do_source(user_vimrc, TRUE, DOSO_VIMRC) == FAIL
```

Comment:
Most of main.c appears to be using TRUE and FALSE. Should I just fix all of them in a new commit/different PR?